tests/expectations/test_expectation_atomic_renderers.py

```diff
@@ -664,7 +664,7 @@ def test_atomic_prescriptive_summary_expect_column_values_to_be_decreasing(
             "column": "my_column",
             "mostly": 0.8,
             "parse_strings_as_datetimes": True,
-            "strictly": 50,
+            "strictly": True,
```

Comment:
This Expectation kwarg does not accept integers. See "Keyword Args" for this Expectation [here](https://greatexpectations.io/expectations/expect_column_values_to_be_increasing?filterType=Backend%20support&gotoPage=1&showFilters=false&viewType=Summary).